homeassistant/components/rotarex/const.py

```diff
@@ -0,0 +1,3 @@
+DOMAIN = "rotarex"
+
+PLATFORMS = ["sensor"]
```

Comment:
This should be moved to init, like this:

https://github.com/home-assistant/core/blob/3b32c4bcbfa4040f994a26437a6f7b80b91fae77/homeassistant/components/anglian_water/__init__.py#L27

You should use the Platform const to define this.